packages/hooks/src/useEventListener.ts

```diff
@@ -44,7 +44,10 @@ export default function useEventListener<Type extends keyof DocumentEventMap>(
       memoizedEventCallback.current(e);
 
     // NOTE(JeT): I'm pretty sure there should be a way to avoid this type assertion, but I couldn't figure it out.
-    element.addEventListener(type, callback as EventListener, options);
+    element.addEventListener(type, callback as EventListener, {
```

Comment:
To correctly removeEventListener do we need to pass in the same options instance?

Reply:
I'm not sure that the instance of the object is required, just that the values are the same.

From the MDN docs, it looks like they don't even do a strict comparison – if I'm reading it right, removeEventListener only cares about the `capture` option's value: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener

Reply:
good call